src/utils/asyncReadLineHandler.js

```diff
@@ -0,0 +1,30 @@
+import readline from 'readline';
+
+/**
+ * 사용자에게 입력값을 받아서, 받은 입력값을 반환하는 함수 입니다.
+ * @param { string } query 사용자에게 안내할 입력 문구
+ * @returns
+ */
+
+function asyncReadLineHandler(query) {
```

Comment:
JSDoc과 선언 사이에는 개행이 없는 것이 일반적이에요.

```suggestion
function asyncReadLineHandler(query) {
```